CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Removed
 
 ### Fixed
+- Fixed handling of NodeInfo fields that are not returned by Amazon OpenSearch Service ([#1131](https://github.com/opensearch-project/opensearch-java/pull/1131))
```

Comment:
Sorry I missed catching this before, it should reference this PR rather than the previous one
```suggestion
- Fixed handling of NodeInfo fields that are not returned by Amazon OpenSearch Service ([#1132](https://github.com/opensearch-project/opensearch-java/pull/1132))
```

Reply:
Oops, good catch! I made the changes.